UDV-Core/src/Modules/Temporal/TemporalModule.js

```diff
@@ -71,7 +73,22 @@ export class TemporalModule {
 
         // Instantiate the temporal window
         // TODO: make it active by default
-        this.temporalWindow = new TemporalWindow(refreshCallback, temporalOptions);
+
+        // load options from baseconfig
+        let n = new NetworkManagerSingleton();
+        n.list_option = temporalOptions.graphOption;
```

Comment:
could you pass it to the constructor instead ? Or create get/set function if it is justified (i.e. you need to set it after the constructor is called)

Reply:
Changed